docs/2.8.0/docs/canton/usermanual/persistence.rst

```diff
@@ -61,6 +61,50 @@ This snippet shows how ssl can be configured for Postgres. You can find more inf
 
 Note that all configuration properties for the database will be propagated to the Ledger API JDBC URL.
 
+.. _postgres-performance-tuning:
+
+Performance 
+~~~~~~~~~~~
+
+Please note that your Postgres database setup will require appropriate tuning to achieve the desired performance. Canton 
+is a rather database heavy process. This section should give you a starting point for your tuning efforts. You may want to consult 
+the :ref:`troubleshooting section <how_to_measure_db_performance>` on how to analyse whether the database is a limiting factor. 
+
+This guide will give you a starting point. Ultimately, every use case is different and the exact resource requirements can not 
+be predicted, but have to be measured.
+
+First, ensure that the database you are using is appropriately sized for your use case. The number of cores depends on your 
+throughput requirements. The rule of thumb is 
+    * 1 db core per 1 participant core.
+    * 1 participant core for 30-100 ledger events (depends on the complexity of the commands).
+
+The memory requirements depend on your data retention period and the size of the data you are storing. Ideally, you monitor the 
+database index cache hit / miss ratio. If your instance needs to keep on loading indexes from the disk, the performance will be 
+poor. It might make sense to start with 128GB, run a long-running scale & performance test and `monitor the cache hit / miss ratio <https://www.postgresql.org/docs/current/monitoring-stats.html#MONITORING-PG-STATIO-ALL-INDEXES-VIEW>`__`
+of the Postgres database. 
+
+Most of the Canton indexes are contract-id based, which means that the index lookups are randomly distributed, for which spinning disks
+have a very poor performance. Use SSDs for your database. 
+
+Optimize the configuration of your database, and make sure the database has sufficient memory and is stored on SSD disks with a very high throughput.
+For Postgres, `this online tool <https://pgtune.leopard.in.ua/>`_ is a good starting point for finding reasonable parameters 
+(use online transaction processing system).
+
+Additionally to the initial configuration, we've made the following observations: Most indexes Canton uses are "hash based".
+Therefore, read and write access to these indexes is uniformly distributed. However, Postgres reads and writes indexes in 
+pages of 8kb, while a simple index might only be a couple of writes. Therefore, it is very important to be able to keep the 
+indexes in memory and only write updates to the disk from time to time, as otherwise, a simple change of 32 bytes requires 8kb IO
+operations. 
```

Comment:
```suggestion
indexes in memory and only write updates to the disk from time to time; otherwise a simple change of 32 bytes requires 8kb IO
operations. 
```

Reply:
fixed